swagger/templates/analysis-inputs.yaml

```diff
@@ -0,0 +1,45 @@
+parameters:
+  - name: resource
+    type: string
+  - name: parameter
+    type: string
+  - name: tag
+    type: string
+template: |
+  parameters:
+    - required: true
+      type: string
+      in: path
+      name: '{{parameter}}'
+    - required: true
+      type: string
+      in: path
+      name: AnalysisId
+  get:
+    summary: Download analysis inputs.
+    description: >
+      If "ticket" query param is included and not empty, download inputs.
+
+      If "ticket" query param is included and empty, create a ticket for all
+      inputs of the anlaysis
+
+      If no "ticket" query param is included, server error 500
+    operationId: download_{{resource}}_analysis_inputs
+    tags: 
+    - '{{tag}}'
+    produces:
+      - application/json
+      - application/octet-stream
+    parameters:
+      - description: ticket id of the inputs to download
+        type: string
+        in: query
+        name: ticket
+    responses:
+      '200':
+        description: ''
+        schema:
+          $ref: schemas/output/analysis-inputs-create-ticket.json
+        examples:
+          response:
+            $ref: examples/output/analysis-inputs-create-ticket.json
```

Comment:
Is there any significant difference between this and `analysis-files.yaml`? Could the differences be paramaterized so that we don't need separate templates for `analysis-inputs.yaml` and `analysis-files.yaml`? Similarly, I'm guessing that the missing `analysis-inputs-create-ticket-filename.yaml` is closely related to `analysis-files-create-ticket-filename.yaml`.

In either case, can we update `analysis-files.yaml` to clarify that these now only include output files?